include/fast_float/ascii_number.h

```diff
@@ -6,16 +6,21 @@
 #include <cstdint>
 #include <cstring>
 #include <iterator>
+#include <version>
```

Comment:
I am not familiar with this header...?

Reply:
`<version>` provides feature test macros such as ` __cpp_lib_bit_cast`, apparently it is not available on older compilers.